src/components/Filters/mocks/options.ts

```diff
@@ -14,7 +14,19 @@ import { Operators } from '../Filters.enums';
 export const fields: Field[] = [
   {
     conditions: [
-      { component: Input, value: 'is', label: 'is' },
+      {
+        component: {
+          component: Input,
+          props: {
+            maxLength: 15,
+            pattern: '[a-zA-Z]+',
+            patternMessage: 'Use only characters',
```

Comment:
This works but I'm just curious if you have considered leaving the validation upon the DT user. Like passing `validate = (values) => { [key]: error }` to the whole <DataTable /> component or individual `validate: requiredFn` as a prop here. 

To me it seems more flexible to pass that responsibility to the user.

Reply:
I understand, it makes sense. I've leverage HTML 5 input validations like pattern, maxLength for type text and min, max for type number. Then evaluation is not in our hands and also error message is browser's native. If we would change it to validation fn, it would require covering also those. Now, I'm wondering how needed change it is

Reply:
Yep, html5 attributes are a good approach and don't want it to sound to delete this straight away 👍 

Things to consider though:
- the only problematic field can be IPs or domains since they can be lists afaik? But that's maybe solvable with some crazy pattern
- global validation would allow making more sophisticated validations - like don't allow filter A if filter B is set (probably not something we want atm but who knows)
- adding a new validation type would require to update DS instead of just updating user implementation
- users can reuse validation function from other places 
- UX would want the error messages/styles custom, wouldn't they?

Reply:
UX would appreciate to have custom styles of messages and also to have it unified with DS forms. I would merge this version, create follow-up and  make unified form validations across whole DS (platform). We will need also UX input how to style it. How does it sounds to you?

Reply:
Sounds good!